flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/debezium/DebeziumJsonFormatFactoryTest.java

```diff
@@ -96,24 +93,39 @@ public void testSeDeSchema() {
                 sinkMock.valueFormat.createRuntimeEncoder(
                         new SinkRuntimeProviderContext(false), PHYSICAL_DATA_TYPE);
 
-        assertEquals(expectedSer, actualSer);
+        assertThat(actualSer).isEqualTo(expectedSer);
     }
 
     @Test
-    public void testInvalidIgnoreParseError() {
-        thrown.expect(
-                containsCause(
-                        new IllegalArgumentException(
-                                "Unrecognized option for boolean: abc. Expected either true or false(case insensitive)")));
-
+    void testInvalidIgnoreParseError() {
         final Map<String, String> options =
                 getModifiedOptions(opts -> opts.put("debezium-json.ignore-parse-errors", "abc"));
 
-        createTableSource(SCHEMA, options);
+        assertThatThrownBy(() -> createTableSource(SCHEMA, options))
+                .satisfies(
+                        anyCauseMatches(
+                                IllegalArgumentException.class,
+                                "Unrecognized option for boolean: abc. "
+                                        + "Expected either true or false(case insensitive)"));
+    }
+
+    @Test
+    void testInvalidOptionForTimestampFormat() {
```

Comment:
please don't change the order of tests; it makes the review unnecessarily difficult